### ISSUE TITLE
[a11y] about page heading levels and landmark issues

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15001]
## Description

There are two a11y issues on the about page related to heading levels and landmark roles.

Heading levels: "A Global View" is an `<h1>`, whilst the city names then jump to `<h3>`.

![image](https://github.com/user-attachments/assets/61ab049e-eb9d-47ef-a271-caf290b312a4)

Landmark roles: the page should have one `<main>` landmark.

### Steps to reproduce

https://www.mozilla.org/en-US/about/

### Expected result

The heading "A Global View" should be an `<h2>`. Wrapping the page content in a `<main>` landmark should fix the other errors (I think).

### Actual result

![image](https://github.com/user-attachments/assets/36e69d50-1b38-4c5b-b312-79a1ca8cb0bd)

### Environment

N/A
